R/derive_merged.R

```diff
@@ -61,7 +61,9 @@
 #'
 #'   The input dataset and the selected observations from the additional dataset
 #'   are merged by the specified by variables. The by variables must be a unique
-#'   key of the selected observations.
+#'   key of the selected observations. Variables from the additional dataset can
+#'   be renamed by naming the element, i.e., `by_vars =
+#'   exprs(<name in input dataset> = <name in additional dataset>)`.
```

Comment:
Should we put something suggesting this is similar to the sql join or dplyr joins??  @bundfussr what do you think?

Reply:
I think the description is pretty clear. But it would not harm to mention the similarity to dplyr.

Reply:
Agreed, don't think there is harm in mentioning it, but I think it'll best be displayed/interpreted through an example.

Reply:
Updated the document to suggest that it's similar to dplyr joins. Do you want an example to be included as well that shows the renaming by_vars capability?